docs/contributing/pull-requests/index.md

```diff
@@ -199,7 +182,24 @@ that may delay this process.
 
 </Community>
 
-### How to perform a review
+### Addressing feedback
+
+It is likely that you will receive some feedback on your PR. You should see this as a positive thing
+-- it signifies a healthy and thorough review process and an organizational commitment to code
+quality. You may receive [comments](./code-review.md#comment) or a
+[request for changes](./code-review.md#request-changes). You are encouraged to engage in
+conversation on the PR to discuss a solution, but if any strong conflicting opinions arise it is
+often best to move the conversation to a synchronous format to avoid any misunderstanding.
+
+When any necessary changes have been made, you should address the comments or request for changes by
+responding in the PR conversation thread. You are not responsible for resolving the conversation --
+that is the prerogative of the reviewer, to ensure that they agree that the question or concern has
+been addressed.
+
+**When you are ready for a reviewer to revisit your changes, you should request a re-review.** This
+will notify the reviewer and ensure a prompt response.
+
+## Performing a review
```

Comment:
This also doesn't fit in with the rest of the page, I recommend it's moved to a callout, maybe at the top of the page. That keeps this page wholly directed on the PR author.

Reply:
This is directed at the author, is it not?  This is telling the PR author to re-request reviews from their reviewers when they have made all requested changes.

Reply:
Sorry, Github is showing additional lines as context which is confusing. This is referring to the `Performing a review` section.

Reply:
I see this now, thank you.  I've added a callout in [`3f8aec1` (#376)](https://github.com/bitwarden/contributing-docs/pull/376/commits/3f8aec1c356cc2c4c35e297f8adcc92ca5baaa69)